client/src/Pages/Game.js

```diff
@@ -9,7 +10,50 @@ import Scores from '../Components/Scores/Scores';
 import Lives from '../Components/Lives/Lives';
 import TimerBar from '../Components/TimerBar/TimerBar';
 
-export default function GamePage() {
+export default function GamePage({ wordData }) {
+  const [
+    selectedWord,
+    setSelectedWord,
+  ] = useState('');
+  const [
+    correctAnswer,
+    setCorrectAnswer,
+  ] = useState('');
+  const [
+    displayAnswer,
+    setDisplayAnswer,
+  ] = useState(true);
+
+  function randomWord() {
+    const randomNumber = Math.floor(
+      Math.random() * wordData.length
+    );
+    const selectedWord =
+      wordData[randomNumber].germanNoun;
+    return setSelectedWord(selectedWord);
+  }
+
+  useEffect(() => randomWord(), []);
+
+  function handleClick(nounGender) {
+    const index = wordData.findIndex(
+      (word) => word.germanNoun === selectedWord
+    );
+    if (wordData[index].gender === nounGender) {
+      console.log('you got 10 points');
```

Comment:
Is it part of the game to log it into the console?

Reply:
Thanks it was just there for me when coding, must have forgotten to delete it.